robot/http/client.robot

```diff
@@ -9,7 +9,7 @@ Test Teardown       Terminate HTTP Server
 
 
 *** Variables ***
-${HTTP_URL}         http://localhost:8000
+${HTTP_URL}         http://localhost:55516
```

Comment:
Are you changing the port number to be consistent with the rest.robot?

Reply:
Consistency, yes, but the test also failed on me because something else was using 8000.